### PR TITLE
NW6 | Zeliha Pala | Full-Stack-Project | Week-2 | DELETE-A- Video-Recommendation

### PR DESCRIPTION
**Pull request by zelihapala**

### CODE REVIEW

**Review by RbAvci — APPROVED** [review]
Well done Zeliha!

File: server/api.js [hunk 1]
```diff
@@ -27,4 +27,19 @@ router.post("/videos", async (req, res) => {
...
+
+	try {
+		await db.query("DELETE FROM videos WHERE id = $1", [videoId]);
+		res
+			.status(200)
+			.json({ success: true, message: "Video deleted successfully" });
+	} catch (error) {
+		res
+			.status(500)
+			.json({ success: false, error: "Failed to delete the video" });
```
Comment (Ara225) [inline server/api.js:41]: Looking good. I appreciate  the use of appropriate HTTP error codes and paramatized queries.

File: client/src/DeleteVideoRecommendation.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+				if (!response.ok) {
+					throw new Error("Failed to delete the video");
+				}
+				return response.json();
+			})
+			.then((data) => {
+				onDelete(videoId);
+			})
+			.catch((error) => {
+				console.error(error);
```
Comment (Ara225) [inline client/src/DeleteVideoRecommendation.jsx:22]: Console.error is not a great solution here. The average user never looks in the console so you should make a habit of using a visible error like a alert()

**Review by Ara225 — COMMENTED** [review]
Good work, one minor gripe

**Pull request merged**